tests/unit/test_progress.py

```diff
@@ -1,19 +1,17 @@
 import logging
-import mock
-from dvc.progress import progress, ProgressCallback
+from dvc.progress import Tqdm
 
 
 def test_quiet(caplog, capsys):
     with caplog.at_level(logging.CRITICAL, logger="dvc"):
-        progress._print("something")
-        assert capsys.readouterr().out == ""
-
-
-class TestProgressCallback:
-    @mock.patch("dvc.progress.progress")
-    def test_should_init_reset_progress(self, progress_mock):
-        total_files_num = 1
-
-        ProgressCallback(total_files_num)
-
-        assert [mock.call.reset()] == progress_mock.method_calls
+        for _ in Tqdm(range(10)):
+            pass
+        out_err = capsys.readouterr()
+        assert out_err.out == ""
+        assert out_err.err == ""
+    with caplog.at_level(logging.INFO, logger="dvc"):
+        for _ in Tqdm(range(10)):
+            pass
+        out_err = capsys.readouterr()
+        assert out_err.out == ""
+        assert "0/10" in out_err.err
```

Comment:
The second with is not about being quiet.

Reply:
I guess this is a sanity check to make sure that tqdm has recovered from the loglevel changed? @casperdcl

Reply:
@efiop indeed, yet again this was my attempt at translating the original test which seemed to be doing the same for the old progress bar.

Reply:
@Suor `assert "0/10" in out_err.err` is not quiet :)

Reply:
ah wait I understand, you don't think the test name (`test_quiet`) matches the task (`assert "0/10" in out_err.err`). Yes it's about both being quiet and recovering. Maybe could rename the test `test_progress_verbosity`, `test_log_level_progress` or similar.

Reply:
Or just remove that code. Why have you add that?

Reply:
The original test did not have that

Reply:
hmm I think it's definitely something which should be tested for (user requesting low verbosity should not get progress bars)

Reply:
and changing verbosity should also re-enable them